### PR TITLE
collaborate updates

### PR DESCRIPTION
**Pull request by maxious**

### CODE REVIEW

File: apps/marketplace/components/Collaborate/Landing.js [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+          <strong className={styles.pop}>
+            <font color="#870f48">
+              {52}
+            </font>
+          </strong>{' '}
+          <article>funded projects</article>
+        </div>
+      </div>
+    </div>
+    <article className={`${styles.filters} row`}>
```
Comment (marcoalfonso) [inline apps/marketplace/components/Collaborate/Landing.js]: Why two rows? needs a column after row

File: apps/marketplace/components/Collaborate/Landing.js [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+                      </div>
+                      <br />
+                      <p>
+                        {val.description}
+                      </p>
+                      <hr />
+                    </div>
+                  )
+                }
+                return undefined
```
Comment (marcoalfonso) [inline apps/marketplace/components/Collaborate/Landing.js]: Is this necesary?

Comment (maxious) [reply apps/marketplace/components/Collaborate/Landing.js]: eslint wants returns
  109:31  error  Expected to return a value at the end of arrow function  array-callback-return
  109:31  error  Expected to return a value at the end of arrow function  consistent-return

File: apps/marketplace/components/Collaborate/Landing.js (hunk 1 above)
Comment (maxious) [reply apps/marketplace/components/Collaborate/Landing.js]: or no rows, updated.

**Pull request merged**